### PR TITLE
CodeforAfrica Error Page CMS setup

### LINKED ISSUE #565: @/codeforafrica Error Pages
Migrate Code for Africa app Error pages to payload CMS.

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR implements the Error page setup in CMS.

Fixes #565 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Why is this still failing @kelvinkipruto ? Any blockers?

**Comment by kelvinkipruto** [comment]
@kilemensi Yes. There is a blocker. We're facing a similar issue for #581. We've identified that it could be caused by the following [change](https://github.com/CodeForAfrica/ui/pull/516/commits/931df37068d8ed4e798a0823cb17fa17ce7eb318) and we're looking into it.

**Comment by kilemensi** [comment]
> @kilemensi Yes. There is a blocker. We're facing a similar issue for #581. We've identified that it could be caused by the following [change](https://github.com/CodeForAfrica/ui/pull/516/commits/931df37068d8ed4e798a0823cb17fa17ce7eb318) and we're looking into it.

That comes straight from the Payload [documentation](https://payloadcms.com/docs/admin/customizing-css#customizing-css-and-scss) @kelvinkipruto so what kind of errors are we getting?

**Comment by kelvinkipruto** [comment]
@kilemensi It was an issue with Payload CSS that caused the previous builds to fail.
image
After some further investigation, we found that it is caused by the DB being empty during the docker build rather than the CSS issue.

File: apps/codeforafrica/src/payload/collections/Pages.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const Pages = {
       // each other e.g. while alphabecially CustomPageHeader should be with C,
       // it's functiaonally equivalent with PageHeader so we keep it next to
       // PageHeader
-      blocks: [Hero, PageHeader, CustomPageHeader],
+      blocks: [Error, Hero, PageHeader, CustomPageHeader],
```
Comment (koechkevin) [inline apps/codeforafrica/src/payload/collections/Pages.js:39]: @kelvinkipruto order this alphabetically

**Review by koechkevin — APPROVED** [review]

**Review by koechkevin — APPROVED** [review]

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**